apps/sms/services/js/drafts.js

```diff
@@ -138,6 +149,13 @@
     },
 
     /**
+     * Returns list of all thread less drafts.
+     */
+    getAllThreadless: function () {
+      return draftIndex.get(null) || [];
```

Comment:
It was failing on new profile since drafts don't exist.... Will check why/if it's not failing on device.